docs/source/models/gbd2017/causes/latent_tuberculosis/index.rst

```diff
@@ -4,6 +4,119 @@
+who do not have rapid progression are classified as having slow-progressing latent
+tuberculosis infection. With latent infection, individuals experience no adverse
+health effects and will not transmit Mycobacterium tuberculosis, but they face
+an ongoing risk of developing active tuberculosis through reactivation.
+
+GBD 2017 Modeling Strategy
+++++++++++++++++++++++++++
+GBD 2017 TB model made a refinement in the mortality-to-incidence (MI) ratio
+regression approach upon 2016 model. The SDI was replaced by Healthcare Access
+and Quality index (HAQI) in the MI ratio regression. First, GBD estimated
+risk-weighted prevalence of LTBI by demographic characteristics using data from
+population-based tuberculin surveys and cohort studies reporting the risk of
+developing active TB disease as a function of induration size. Next, they
+divided the inputs on prevalence, incidence, and cause-specific mortality rate
+(CSMR) by the risk-weighted LTBI prevalence in order to model TB among at risk
+populatoin. GBD ran a mixed effects regression (with region random effects)
+using logit transformed MI ratios from locations with a 4-star or above rating
+(countries with poor surveillance systems are classified as less than 4 stars)
+on causes of death with HAQI as a covariate anchoring the lower bound of the
+HAQI scale with a data point reported in the Bangalore study to predict age-sex
+specific MI ratios for all locations and years. Then the age-sex-specific
+incidence were estimated based on those predicted MI ratios. GBD used DisMod-MR
+2.1 (Bayesian meta-regression tool) to generate consistent trends in all
+parameters, and multiplied the outputs by the risk-weighted prevalence of LTBI
+to get population-level estimates of incidence and prevalence. GBD disaggregated
+the all-form TB outputs from DisMod-MR 2.1 into MDR-TB and XDR-TB by HIV status.
+To do so, they estimated the proportions of TB cases with MDR-TB for all
+locations and years, and then estimated the proportions of MDR-TB among
+HIV-negative and HIV-positive individuals based on the risk of MDR-TB associated
+with HIV infection from a meta-analysis. To split MDR-TB into MDR-TB with and
+without extensive drug resistance, GBD pooled the notification and survey data
+on the proportion of MDR-TB cases with extensive drug resistance by
+super-region, and applied these proportions to MDR-TB cases among HIV-negative
+and HIV-positive individuals, respectively. [GBD-2017-YLD-Capstone-Appendix-1]_
+
+Summary model description
++++++++++++++++++++++++++
+It's a complex joint model that includes both tuberculosis and HIV/AIDS. We
+model TB and HIV together because HIV has impacts on both progression rate of
+LTBI to active TB and active TB excess mortality rate. Note that we have
+excluded the active TB drug resistance and GBD predicted the proportion of new
+TB cases with MDR-TB or XDR-TB by ST-GPR. In our model, each simulant occupies
```

Comment:
Can you clarify if you excluded MDR-TB and XDR-TB cases entirely, or if you've lumped them in with your TB-prevalent cases?

Reply:
When I say exclude MDR-TB and XDR-TB, it actually means we only split active TB by HIV status, not drug resistance.